### PR TITLE
GEOPY-1526: Update and rename bat files

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1526 - Update bat file**

### CODE REVIEW

File: install_for_analyst.bat [hunk 1]
```diff
@@ -7,4 +7,4 @@
 
 set "MIRA_CMD_RUNNER_DIR=%ProgramFiles%\Mira Geoscience\Geoscience ANALYST\CmdRunner"
 set curdir=%~dp0
-"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --install "%curdir%\environments\conda-py-3.10-win-64.lock.yml" --with-pip-deps %*
+"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --install "%curdir%\environments\conda-py-3.10-win-64.lock.yml" --with-pip-deps %* --run pip install -e .[dash] %*
```
Comment (sebhmg) [inline install_for_analyst.bat:10]: I had hacked a quick thing  to install petro-lingo all from pip under a new env for Analyst,
but that does not quite apply here.

Command is more:
```suggestion
"%MIRA_CMD_RUNNER_DIR%\MambaEnvRunner.exe" --name peak-finder --install "%curdir%environments\py-3.10-win-64.conda.lock.yml" ^
  --run "pip install -e .[dash]" %*
```

Comment (sebhmg) [reply install_for_analyst.bat:10]: update app in the first line comment:
```bat
:: Creates an environment for peak-finder-app under the Python environment location used by Geoscience Analyst.
```

**Review by sebhmg — CHANGES_REQUESTED** [review]
`install_in_mirageo,bat` is not quite correct

**Review by sebhmg — APPROVED** [review]

File: install.bat [hunk 2]
```diff
@@ -17,15 +17,15 @@ set PYTHONUTF8=1
...
 if not exist %MY_CONDA_ENV_FILE% (
   echo "** ERROR: Could not find the conda environment specification file '%MY_CONDA_ENV_FILE%' **"
   pause
   exit /B 0
 )
 
 call "!MY_CONDA!" activate base ^
-  && call "!MY_CONDA!" env create --force -n %ENV_NAME% --file %MY_CONDA_ENV_FILE% ^
+  && call "!MY_CONDA!" env create -y --solver libmamba -n %ENV_NAME% --file %MY_CONDA_ENV_FILE% ^
   && call "!MY_CONDA!" run -n %ENV_NAME% pip install -e .[dash]
```
Comment (sebhmg) [inline install.bat:29]: for users, I would not make it editable
```suggestion
  && call "!MY_CONDA!" run -n %ENV_NAME% pip install .[dash]
```

As for developers, we could have a helper script `make-editable.bat` to patch the installation and point to the local source.

**Review by sebhmg — CHANGES_REQUESTED** [review]
my recommendation would be to make installation non-editable for end users. Possibly have a helper script to turn it into editable installation for developers.

(Ideally, installation should not require users to download the repo. Will address that when reworking on the management of conda and pip dependencies)

**Comment by sebhmg** [comment]
@domfournier , I will PR on the this PR branch with a suggestion of scripts (non-editable, patch as editable)

**Review by benk-mira — APPROVED** [review]

**Review by sebhmg — APPROVED** [review]

**Pull request merged**